core/src/main/java/io/questdb/std/IntList.java

```diff
@@ -120,7 +120,6 @@ public int getLast() {
      * @return element at the specified position.
      */
     public int getQuick(int index) {
-        assert index < pos;
```

Comment:
i think this method was designed not to check the index, at least that is what the javadoc above the method says.
Problem is that get(index) just calls getQuick(index) so get will not check index boundaries either.
I think this method is ok without the assert but instead of completely removing it we should move it into get().

Reply:
I think it is my mistake. I assumed that we’d get array index out of bounds when index is bad and this assert to be duplicate. But we over-allocate the array so this assert highlights issues in code when array index out of bounds does t happen. We should reinstate the assert. I don’t think there is performance impact because asserts can be disabled

Reply:
ok, then i will just add the assert back